plugins/lsp/language-servers/default.nix

```diff
@@ -238,6 +238,12 @@ let
       serverName = "java_language_server";
       cmd = cfg: [ "${cfg.package}/bin/java-language-server" ];
     }
+    {
+      name = "jdt-language-server";
+      description = "Eclipse JDT Language Server for Java";
+      serverName = "jdt_language_server";
```

Comment:
The CI error means this line is wrong, we should reference the nvim-lspconfig docs to see what it should be.

Reply:
Thanks, I will look deeper into it. I was having trouble running the test suite locally as it targets all architectures. Is there an easy way to emulate that or something? I am on NixOS 24.05

Reply:
Looks like they drop the whole name in favor of `jdtls`, https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#jdtls

Reply:
> Thanks, I will look deeper into it.

Brilliant 👍 

> I was having trouble running the test suite locally as it targets all architectures.

It's not important to run all systems locally, it's fine to let CI handle that.

Usually you just need the `tests` command made available in the devshell (`nix develop`).